### PR TITLE
Unit 3.2 linting 

### LINKED ISSUE #13: 3.2: Linting
# 3.2: Linting
## Main Goal
Add JavaScript linting to the project to ensure formatting and style standards are being met.
### When Accomplished
- [x] The linter runs on commands `yarn start` and `yarn build`
- [x] Linting completes with no errors
## Research
- [How Linting an ESLint Improve Code Quality - HackerNoon](https://hackernoon.com/how-linting-and-eslint-improve-code-quality-fa83d2469efe)
## Resources
- [ESLint](https://eslint.org/)
- [ESLint: Rules Reference](https://eslint.org/docs/rules/)
- [Banno/ux-lint - GitHub](https://github.com/Banno/ux-lint)

### PR DESCRIPTION
**Pull request by frankiebarrios**
## What It Does
This pull request is set to complete Unit 3.2 of the ux-onboarding-project.

This work is taking place to complete unit 3.2 of the ux-onboarding-project.

**Implementation:**
- Added Ux-Lint as linter.
- Added scripts to run linter.
- Updated README with further instructions.

[closes #13 ]

## How To Test
- Install dependencies using instruction in README.
- Start server using yarn start or build using yarn build.

## Notes/Caveats

None
## Checklist

Under penalty of public shaming, I avow that I:
**Many items in list N/A** 
- [X] Reviewed the diff to look for typos, whitespace errors, and console.log() statements
- [X] Verified that there are no compiler or linter errors or warnings
- [X] Verified the build in production(optimized) mode
- [X] Updated the docs, if necessary
- [X] Included the appropriate labels
- [X] Referenced applicable issues (i.e. Closes #XXXX, Fixes #XXXX)

Browsers tested:
**Many items in list N/A** 
- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] Internet Explorer

## Dependencies
1. Install Node and npm
2. Install Yarn
3. Install Ux-Lint

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "description": "UX Team Intern Onboarding Project",
   "main": "index.html",
   "scripts": {
-    "start": "http-server ./src -p 1820"
+    "lint": "ux-lint --fix",
```
Comment (kddrake) [inline package.json:7]: this errors out... you'll need to have `ux-lint` in your dev deps

File: package.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
...
+    "start": "yarn lint && http-server ./src -p 1820",
+    "build": "yarn start"
```
Comment (kddrake) [inline package.json]: this honestly would be the other way around, I'd expect `yarn start` to run `yarn lint`, then` yarn build` then the serve command (maybe make that it's own script?)

**Review by kddrake — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (frankiebarrios) [reply package.json]: Got it, anything specific that has to happen within build at this step in the project?

Comment (kddrake) [reply package.json]: not really, I'm not entirely sure why I included that to begin with. But here we are 🤷‍♂

Comment (frankiebarrios) [reply package.json]: Lol ok

**Review by kddrake — APPROVED** [review]

**Pull request merged**